### PR TITLE
[API-8527] update docs

### PR DESCRIPTION
**Pull request by thehung111**

### CODE REVIEW

File: src/main/java/com/visenze/productsearch/README.md [hunk 1]
```diff
@@ -147,7 +139,7 @@ ProductSearch api = new ProductSearch.Builder(APP_KEY, PLACEMENT_ID)
                      
 // Use the ProductSearch object to perform searches, it will return a
 // ProductSearchResponse object that contains all the searched results info 
-ProductSearchResponse searchResults = api.visualSimilarSearch(params);  
+ProductSearchResponse searchResults = api.recomendation(params);
```
Comment (srfication) [inline src/main/java/com/visenze/productsearch/README.md:142]: looks like typo. should be `recommendation`? missing an `m`

Comment (thehung111) [reply src/main/java/com/visenze/productsearch/README.md:142]: unfortunately it is the code typo, i can fix but then need to rename and may break backwards compatibility.

**Review by srfication — APPROVED** [review]

File: src/main/java/com/visenze/productsearch/README.md (hunk 1 above)
Comment (srfication) [reply src/main/java/com/visenze/productsearch/README.md:142]: ic ok

Comment (thehung111) [reply src/main/java/com/visenze/productsearch/README.md:142]: https://github.com/visenze/visearch-sdk-java/blob/master/src/main/java/com/visenze/productsearch/ProductSearch.java#L211

**Pull request merged**